sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataSourceCredentialEntityLiveTests.cs

```diff
@@ -22,11 +22,11 @@ public DataSourceCredentialEntityLiveTests(bool isAsync) : base(isAsync)
         }
 
         [RecordedTest]
-        [TestCase(nameof(DataSourceDataLakeGen2SharedKey))]
-        [TestCase(nameof(DataSourceServicePrincipal))]
-        [TestCase(nameof(DataSourceServicePrincipalInKeyVault))]
-        [TestCase(nameof(DataSourceSqlConnectionString))]
-        public async Task CreateAndGetDataSourceCredential(string credentialTypeName)
+        [TestCase(nameof(DataLakeSharedKeyCredentialEntity))]
+        [TestCase(nameof(ServicePrincipalCredentialEntity))]
+        [TestCase(nameof(ServicePrincipalInKeyVaultCredentialEntity))]
+        [TestCase(nameof(SqlConnectionStringCredentialEntity))]
+        public async Task CreateAndGet(string credentialTypeName)
```

Comment:
Recording path + filenames were exceeding the limit imposed by Windows, so decided to reduce the redundancy in test names.

Reply:
it is more like a plop face :P